libtransmission/peer-io.cc

```diff
@@ -1071,7 +1050,7 @@ static int tr_peerIoTryWrite(tr_peerIo* io, size_t howmuch)
         tr_logAddDebugIo(io, fmt::format("unsupported peer socket type {}", io->socket.type));
     }
 
-    return n;
+    return n_bytes;
```

Comment:
why not returning `n` like before?
Especially in case of error, it's probably best to return `n` than n_bytes.

Reply:
Agreed, this should not have been changed. Reverted in a5d7768ed3508c5186556080e24899be991beb79